### DISCUSSION TITLE
How to use plugin except on certain file types?

### DISCUSSION THREAD

**Discussion by riazufila** [wbthomason/packer.nvim#594] [Q&A]
The following example use `ale` on filetypes `sh`, `zsh`, ..., `tex`.
```
use {
    'w0rp/ale',
    ft = {'sh', 'zsh', 'bash', 'c', 'cpp', 'cmake', 'html', 'markdown', 'racket', 'vim', 'tex'},
    cmd = 'ALEEnable',
    config = 'vim.cmd[[ALEEnable]]'
  }
```
Is there an option to exclude `githubname/plugin` on certain file type? Such that:

```
use {
    'githubname/plugin',
    exclude_ft = {'tex'}
  }
```

**Comment by wbthomason** [comment]
We do not currently have a way to avoid loading a plugin on a filetype, but this could be added - the gist of the approach would be adding a `FileType *` autocommand that checks the filetype against a blocklist to determine if we should load the plugin or not. I'd be happy to guide/review a PR if you're interested!
